wpilib/wpilib/cantalon.py

```diff
@@ -168,6 +168,7 @@ def handle(self):
         return self._handle
 
     def free(self):
+        LiveWindow.removeComponent(self)
```

Comment:
In fact, if the dictionary is holding references to the object, then the finalizer will never run anyways. Perhaps it should be a dictionary of weak references.